go/uidmap/uidmap.go

```diff
@@ -299,7 +299,7 @@ func (u *UIDMap) MapUIDsToUsernamePackages(ctx context.Context, g libkb.UIDMappe
 		//
 		// Thus, if you provide forceNetworkForFullName=false, and fullNameFreshness=0, you can avoid
 		// the network trip as long as all of your username lookups hit the cache or are hardcoded.
-		if up == nil || (status == notFound && forceNetworkForFullNames) || (status == stale) {
+		if up == nil || ((status == notFound || up.FullName == nil) && forceNetworkForFullNames) || (status == stale) {
```

Comment:
Also cc @mmaxim this seems to have fixed my standalone issue. It also sometimes happened during normal daemon mode, I couldn't pinpoint it to anything though (nuking db didn't seem to change anything for better or worse). From my limited understanding of this code, it seemed like `findUsernamePackageLocally` was returning <nil> FullName and OK status despite `forceNetworkForFullNames` being true. It looks like `findUsernamePackageLocally` doesn't even check for `forceNetworkForFullNames`.

Reply:
or maybe ccing wrong person, it looks like mostly @maxtaco code!

Reply:
Don't think this is the right play here, this interface is supposed to be fine with no full name.